_data/download_platforms.yml

```diff
@@ -24,64 +24,74 @@
 
 - name: "android.playstore"
   title: "Android"
-  caption: "Play Store Universal"
+  caption: "Play Store"
   tags:
     - Play Store
     - arm64
     - arm32
     - x86_64
     - x86_32
 
+- name: "android.horizonos"
+  title: "Horizon OS"
+  caption: "APK - arm64"
+  tags:
+    - APK download
+    - Meta Quest 3 & Pro
+    - arm64
+
+- name: "android.horizonstore"
+  title: "Horizon OS"
+  caption: "Horizon Store"
+  tags:
+    - Horizon Store
+    - Meta Quest 3 & Pro
+    - arm64
+
 - name: "linux.32"
   title: "Linux"
-  caption: "Standard (x86_32)"
```

Comment:
I removed the "Standard" which didn't say anything useful. It's normally used to distinguish between "Standard" (no C#) and ".NET" (C#) builds, but the way this is designed, .NET builds were actually listed as "Standard - .NET" so all builds were "Standard".